temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for debugging or logging. Use the logging module instead to provide more control over logging levels and outputs.
```
import logging

# Setup logging configuration
logging.basicConfig(level=logging.INFO)

# Example usage
logging.info('hello')
```